### PR TITLE
feat: Add settings for boost 

### PR DESCRIPTION
**Pull request by samuveth**
### Summary

image

### CODE REVIEW

File: src/composables/useFormSpaceSettings.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ const EMPTY_SPACE_FORM = {
     type: '',
     privacy: ''
   },
+  boost: {
+    enabled: false,
```
Comment (ChaituVR) [inline src/composables/useFormSpaceSettings.ts:37]: will be enabled by default no?

```suggestion
    enabled: true,
```

File: src/views/SpaceBoost.vue
Comment (ChaituVR) [inline src/views/SpaceBoost.vue:1]: I think we should either hide this component or disable create button if not whitelisted, for example 
https://snapshot-git-samuv-boost-settings-snapshot.vercel.app/#/ens.eth/boost/0xd3437f1c9ece8a309da116be5bffaf31fa40df5361e04e43f9c913970a8746ee

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Review by ChaituVR — APPROVED** [review]
tAck

File: src/views/SpaceBoost.vue
Comment (samuveth) [reply src/views/SpaceBoost.vue:1]: Done @ChaituVR 
image

Comment (ChaituVR) [reply src/views/SpaceBoost.vue:1]: Perfect!

**Pull request merged**